app/assets/javascripts/application/rvt_helper.js

```diff
@@ -0,0 +1,46 @@
+function removeRvtHelper(userId) {
+    const url = `/admin/rvt_helpers/${userId}/remove`
+    fetch(url, {
+        method: 'post',
+        headers: {'X-CSRF-Token': getCsrfToken()}, // this line is important, if this content-type is not set it wont work
+    }).then(function (response) {
+        return response.text();
+    }).then(function (result) {
+        removeRvtHelperFromList(userId);
+        removeRvtHelperButton(userId);
+    }).catch(function (err) {
+        console.warn('Something went wrong.', err);
+    });
+}
+
+function removeRvtHelperFromList(userId) {
+    const element = document.getElementById(`rvt-helper-user-${userId}`);
+    if (element) {
+        element.remove()
+    }
+}
```

Comment:
This is the same logic as `removeRvtHelperButton`, maybe it's worth extracting to a common function.